src/components/Button.jsx

```diff
@@ -0,0 +1,6 @@
+import style from './Button.module.css';
+
+function Button({ children, size, variant }) {
```

Comment:
Could you please add a comment above the component describing what are the available options for `size` and `variant`. Since we don't use TypeScript nor PropTypes it's unclear what can we use.

Reply:
Also, I guess we need default values for both - variant and size.